recipes/rangesnext/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class RangesnextConan(ConanFile):
+    name = "rangesnext"
+    description = "ranges features for C++23 ported to C++20"
+    topics = ("conan", "rangesnext", "ranges", "backport", "backport-cpp")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cor3ntin/rangesnext"
+    license = "BSL-1.0"
+    exports_sources = "patches/**"
+    settings = "compiler"
```

Comment:
```suggestion
    settings = "compiler"
    no_copy_source = True
```
The documentation need to be update, but Conan copies from self.source_folder to self.build_folder by default. However, this recipe is too simple, we don't need build anything, only do a copy from self.source_folder to self.package_folder.

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#no-copy-source